python/ray/experimental/state/state_cli.py

```diff
@@ -32,16 +31,6 @@
 logger = logging.getLogger(__name__)
 
 
-def _alpha_doc():
```

Comment:
@rickyyx I removed it because we can add this information to the top-level document (and it also screws up the formatting of docstring for some reasons. Tried fixing it, but no luck)